src/services/EcoBridge/Connext/Connext.lists.ts

```diff
@@ -0,0 +1,211 @@
+        contract_decimals: 8,
+      },
+      [ChainId.ARBITRUM_ONE]: {
+        contract_address: '0x2f2a2543b76a4166549f7aab2e75bef0aefc5b0f',
+        contract_decimals: 8,
+      },
+      [ChainId.XDAI]: {
+        contract_address: '0x8e5bbbb09ed1ebde8674cda39a0c169401db4252',
+        contract_decimals: 8,
+      },
+      [ChainId.POLYGON]: {
+        contract_address: '0x1bfd67037b42cf73acf2047067bd4f2c47d9bfd6',
+        contract_decimals: 8,
+      },
+    },
+  },
+  {
+    id: 'grt',
+    symbol: 'GRT',
+    name: 'Graph Token',
+    logoURI: 'https://maticnetwork.github.io/polygon-token-assets/assets/grt.svg',
+    contracts: {
+      [ChainId.MAINNET]: {
+        contract_address: '0xc944e90c64b2c07662a292be6244bdf05cda44a7',
+        contract_decimals: 18,
+      },
+      [ChainId.ARBITRUM_ONE]: {
+        contract_address: '0x23a941036ae778ac51ab04cea08ed6e2fe103614',
+        contract_decimals: 18,
+      },
+      [ChainId.XDAI]: {
+        contract_address: '0xfadc59d012ba3c110b08a15b7755a5cb7cbe77d7',
+        contract_decimals: 18,
+      },
+      [ChainId.POLYGON]: {
+        contract_address: '0x5fe2b58c013d7601147dcdd68c143a77499f5531',
+        contract_decimals: 18,
+      },
+    },
+  },
+  {
+    id: 'gno',
+    symbol: 'GNO',
+    is_staging: true,
+    logoURI: 'https://maticnetwork.github.io/polygon-token-assets/assets/gno.svg',
+    name: 'Gnosis Token',
+    contracts: {
+      [ChainId.MAINNET]: {
+        contract_address: '0x6810e776880c02933d47db1b9fc05908e5386b96',
+        contract_decimals: 18,
+      },
+      [ChainId.ARBITRUM_ONE]: {
+        contract_address: '0xa0b862f60edef4452f25b4160f177db44deb6cf1',
+        contract_decimals: 18,
+      },
+      [ChainId.XDAI]: {
+        contract_address: '0x9c58bacc331c9aa871afd802db6379a98e80cedb',
+        contract_decimals: 18,
+      },
+      [ChainId.POLYGON]: {
+        contract_address: '0x5ffd62d3c3ee2e81c00a7b9079fb248e7df024a8',
+        contract_decimals: 18,
+      },
+    },
+  },
+  {
+    id: 'fei',
+    symbol: 'FEI',
+    name: 'Fei USD',
+    contracts: {
+      [ChainId.MAINNET]: {
+        contract_address: '0x956f47f50a910163d8bf957cf5846d573e7f87ca',
+        contract_decimals: 18,
+      },
+      [ChainId.ARBITRUM_ONE]: {
+        contract_address: '0x4a717522566c7a09fd2774ccedc5a8c43c5f9fd2',
+        contract_decimals: 18,
+      },
+      [ChainId.POLYGON]: {
+        contract_address: '0xc7031408c7978da9aca03308cd104cb54e7a2eb3',
+        contract_decimals: 18,
+      },
+    },
+  },
+  {
+    id: 'magic',
+    symbol: 'MAGIC',
+    is_staging: true,
+    name: 'Magic',
+    contracts: {
+      [ChainId.MAINNET]: {
+        contract_address: '0xb0c7a3ba49c7a6eaba6cd4a96c55a1391070ac9a',
+        contract_decimals: 18,
+      },
+      [ChainId.ARBITRUM_ONE]: {
+        contract_address: '0x539bde0d7dbd336b79148aa742883198bbf60342',
+        contract_decimals: 18,
+      },
+    },
+  },
+]
```

Comment:
Can we import the ones we have from constants?

Reply:
done [e7abced](https://github.com/levelkdev/swapr-dapp/pull/1008/commits/e7abcedec1a562e0bb86a0c8324031aa65958c6c)